force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -594,8 +594,8 @@
         <fullName>Only_Showing_Future_Sessions</fullName>
         <language>en_US</language>
         <protected>true</protected>
-        <shortDescription>Only showing sessions on or after today&apos;s date</shortDescription>
-        <value>Only showing sessions after today&apos;s date</value>
+        <shortDescription>Note about sessions displayed when adding sessions.</shortDescription>
+        <value>Showing all future Sessions. To modify previously added Sessions, go to the Service Sessions list.</value>
```

Comment:
@jjbennett, Shouldn't this say 'Showing all upcoming Sessions.' instead of 'Showing all future Sessions.'

Reply:
I spoke with Jen on this, since we wanted to let the user know that past sessions would show we went back to the word "future"